### ISSUE TITLE
make `renderInto` a plain method

### ISSUE THREAD

**Issue by armanbilge** [armanbilge/calico#82]
suggestion from @kubukoz

> personally I wouldn't mind if renderTo was a plain method instead of an extension
generally if you only ever call this once, it's not worth the mental overhead in my book
> just saying it might be more user-friendly if that's a default

**Comment by hejfelix** [comment]
Yes. Autocompletion also seems to have a hard time with it in its current state.

**Comment by kubukoz** [comment]
To be fair, on non-extensions you get no dot completions whatsoever.

**Comment by armanbilge** [comment]
Btw, extension methods can also be used as ordinary methods.

https://github.com/armanbilge/calico/blob/296274912d750a74cd69b166a837a04c8a3cbef8/calico/src/main/scala/calico/syntax.scala#L46-L48

So you may use it as either:
- `component.renderInto(node)`
- `renderInto(component)(node)`

**Comment by kubukoz** [comment]
true, I forgot this was Scala 3 only :)

**Comment by armanbilge** [comment]
> generally if you only ever call this once

At the time I think you also didn't know about `IOWebApp` :) `renderInto` is basically like `allocated` or `unsafeRun*, something hopefully not used at all.

**Comment by kubukoz** [comment]
yeah, that's a good point :D
